interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/parameters-overview.md

```diff
@@ -4,75 +4,95 @@ author: Anna Gubareva
+
+* **Show in the parameters panel** (corresponds to the parameter's **Visible** property)  
+	Enable this option to request the parameter value in Print Preview. Otherwise, the report takes the default parameter value.
+* **Allow multiple values** (corresponds to the parameter's **Multi-Value** property)  
+	Enable this option to allow a parameter to accept a [collection of values](create-multi-value-and-cascading-parameters.md).
+* **Allow null value** (corresponds to the parameter's **Allow Null** property)  
+	Enable this option if the parameter's value can be unspecified.
+* **Range Value**  
+	Enable this option if the parameter should specify a range with a start and end value. This option applies to _Date_-type parameters. See [Date Range Parameters](date-range-parameters.md) for information on how to configure a date range parameter.
+* **Supports the collection of standard values**  
+	Applies if the parameter is visible (its value is requested in Print Preview). You can choose a value from a predefined list. You can populate this list with static values, or specify a data source from where the values are obtained.
 
-* **Name** - specifies the unique name by which the parameter can be referred to.
-* **Description** - specifies the text that will be displayed in a Print Preview along with the corresponding value editor.
-* **Type** - specifies the parameter's value type, according to which an appropriate value editor is displayed in a Print Preview.
-* **Default value** - specifies the default parameter value.
-* **Show in the parameters panel** (corresponds to the parameter's **Visible** property) - enable this option to request the parameter value in a Print Preview. Otherwise, the default parameter value is silently passed to the report.
-* **Supports the collection of standard values** - you can enable this option if the parameter is visible (i.e., its value should be requested in a Print Preview). In this case, you can choose a value from a predefined list. You can either manually populate this list with possible values, or specify a data source from where these values should be obtained.
-	
 	* **Dynamic values**
-		
-		On this tab, you can specify a data source, data adapter (if required) and data member storing parameter values. The value member defines a data field that will provide values to the parameter. The display member defines a data field storing values displayed in a Print Preview.
+
+		Specify a data source, data adapter, and data member for the parameter values storage. **Value Member** defines the data field that provides the parameter's values. **Display Member** defines the data field that stores values displayed in Print Preview.
 		
 		![](../../../../../images/eurd-win-parameters-dynamic-values.png)
 		
-		The value type of the specified data member should match the specified parameter type.
+		The specified data member's value type should match the specified parameter type.
```

Comment:
```suggestion
		The data member's value type should match the parameter type.
```

Reply:
I'm getting rid of the word 'should'. So I changed that to:
If the data member's value type does not match the parameter type, the validation rejects the value.